content/millennium/r4/conformance/structure-definition.md

```diff
@@ -69,21 +69,25 @@ Authorization is not required.
-[`financial-transaction-type`]            |  The classification of the transaction.
-[`account-related-parts`]                 |  A reference to other related Accounts.
-[`account-state`]                         |  The status of the Account within the billing or correspondence workflow.
+ID                                                          |  Description
+------------------------------------------------------------|----------------------------------------------
+[`account-balance`]                                         |  Represents the account balance.
+[`financial-transaction-account-number`]                    |  A value associated to the specific payment method usually represented as the last four digits of a credit card, the check number, the EFT number or Lockbox number.
+[`financial-transaction-alias`]                             |  Client defined value to represent the combination of the type, subtype, and reason describing the financial transaction.
+[`financial-transaction-allocation`]                        |  Defines how the payment or adjustment is to be allocated across other resources.
+[`financial-transaction-amount`]                            |  The total amount of the financial transaction.
+[`financial-transaction-card-brand`]                        |  Identifies the brand of credit card when credit card is used as a payment method.
+[`financial-transaction-date`]                              |  Represents the expiration date if method is card, check date if method is check, EFT date if method is EFT or Lockbox date if method is lockbox.
+[`financial-transaction-location`]                          |  Client configured value representing the location or workflow that the payment was received in.
+[`financial-transaction-method`]                            |  Describes the method of payment for the financial transaction.
+[`financial-transaction-tendered-amount`]                   |  The amount of cash originally tendered for payment. This value should be greater than or equal to the amount of the cash payment.
+[`financial-transaction-type`]                              |  The classification of the transaction.
+[`account-related-parts`]                                   |  A reference to other related Accounts.
+[`account-state`]                                           |  The status of the Account within the billing or correspondence workflow.
+[`client-organization`]                                     |  The financially responsible organization.
+[`estimated-financial-responsibility-amount`]               |  The estimated amount to be collected for the encounter.
+[`payment-collection-status`]                               |  The status of the payment collection for the encounter.
+[`estimated-financial-responsibility-not-collected-reason`] |  The reason no estimated amount is collected for the encounter.
```

Comment:
i guess we should alphabetize these

Reply:
https://github.com/cerner/fhir.cerner.com/pull/415/commits/346d1c84ec066ff89fd8f5dfdb7fec58712cc176#diff-56528e5fbe9efbbf400be5f023a9a599R74